conferences/abstracts/RSECon19-BinderHub-workshop-application.md

```diff
@@ -0,0 +1,151 @@
+# Research Software Engineering Conference 2019
+
+* Conference website: https://rse.ac.uk/conf2019/
+* Application form: https://docs.google.com/forms/d/e/1FAIpQLScVml19n44QfuOfjuaRlJUtXNMih-X4xUrNjlr8XJVHeyTyQg/viewform
+* Location: University of Birmingham
+* Date: 17th - 19th September 2019
+
+### Workshop Length
+
+180 mins (to allow time for Kubernetes cluster deployment and an introduction to reproducible research and BinderHub).
+
+### Title
+
+"Build a BinderHub for hosting Reproducible Software in the Cloud"
+
+### Authors
+
+* [Sarah Gibson](https://www.turing.ac.uk/people/researchers/sarah-gibson), ([The Alan Turing Institute](https://www.turing.ac.uk/)) on behalf of **[The Turing Way](https://github.com/alan-turing-institute/the-turing-way) Collaboration**.
+
+### Abstract
+
+_(1250 characters max, current count: 1244 [please update if you edit])_
+
+BinderHub is a Cloud-based, multi-user server technology that captures the computing environment of a GitHub repository within a Docker image.
+It then provides an interactive browser from which code can be run in the environment under which it was developed without placing installation responsibilities on the user.
+A unique URL to this browser is generated making the software easily sharable with collaborators.
+BinderHubs can be built anywhere in the world and are Cloud-neutral.
+Building a locally hosted BinderHub is advantageous as it would facilitate the use of private repositories and make available a greater variety of computing resources - facilities which are not currently available on the public Binder service.
```

Comment:
The very first time I read this my mind read "locally hosted" as "running on my laptop". By the time you read to the end of the sentence it is clear that "local" means "local to your institute/organisation". I can't think of a quick fix but maybe someone else has a good idea on how to phrase it so your mind doesn't go "wut, on my laptop?

Reply:
"Building an in-house BinderHub..."?